generatedTypes/services/LogService.d.ts

```diff
@@ -1,11 +0,0 @@
-declare function warn(message: string): void;
-declare function deprecationWarn({ component, oldProp, newProp }: {
-    component: string;
-    oldProp: string;
-    newProp?: string;
-}): void;
-declare const _default: {
-    warn: typeof warn;
-    deprecationWarn: typeof deprecationWarn;
-};
-export default _default;
```

Comment:
This file is actually required.. 
run `npm run build:dev` and you'll see this file added  :)